features/FEATURE_BLE/source/generic/GenericGap.cpp

```diff
@@ -1210,6 +1206,8 @@ ble_error_t GenericGap::startRadioScan(const GapScanningParams &scanningParams)
         return err;
     }
 
+    _scan_enabled = true;
```

Comment:
We don't care much about V2 event handling with function belonging to V1 API.

Reply:
required for stop to happen